src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contact-selection/main.js

```diff
@@ -0,0 +1,420 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * Customer content type.
+ *
+ * Allows selection of multiple contacts.
+ */
+define([], function() {
+
+    'use strict';
+
+    var defaults = {
+            eventNamespace: 'sulu.contact-selection',
+            resultKey: 'customers',
+            contactResultKey: 'contacts',
+            accountResultKey: 'accounts',
+            dataAttribute: 'contact-selection',
+            dataDefault: [],
+            hidePositionElement: true,
+            hideConfigButton: true,
```

Comment:
Are these two settings really used? Asked in a different way, does it make sense to change them for this content type?